lib/linguist/languages.yml

```diff
@@ -7861,6 +7894,33 @@ mcfunction:
   tm_scope: source.mcfunction
   ace_mode: text
   language_id: 462488745
+Pencil:
+  type: prose
+  color: "#ff6633"
+  aliases:
+    - mdp
+    - penc
+    - pencil
+    - xpro
+    - mdpro
+  ace_mode: markdown
+  codemirror_mode: gfm
+  codemirror_mime_type: text/x-gfm
+  wrap: true
+  extensions:
+    - ".mdp"
+    - ".penc"
+    - ".pencil"
+    - ".xpro"
+    - ".mdpro"
+  filenames:
+    - style1.mdp
+    - style2.mdp
+    - style3.mdp
+    - example.mdp
+    - example.mdp
```

Comment:
```suggestion

```
If the extension is `mdp` then you dont need to manually specify any filenames that also have that extension

Reply:
.pencil, fixed on next pull request